sockutils.c

```diff
@@ -1004,7 +1031,9 @@ int sock_recv(SOCKET sock, SSL *ssl _U_NOSSL_, void *buffer, size_t size,
 	 * Win32.
 	 */
 	for (;;) {
-#ifdef HAVE_OPENSSL
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+		nread = fuzz_recv(bufp, remaining);
+#elif defined(HAVE_OPENSSL)
 		if (ssl)
```

Comment:
You can't remove `#ifdef HAVE_OPENSSL` - this has to build even without OpenSSL.

Reply:
It is not removed, it is moved 3 lines below.
fuzz socket now overrides both ssl and non-ssl sockets